packages/contact-manager-rest-api/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@sphereon/ssi-sdk.contact-manager-rest-api",
+  "version": "0.15.1",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --build",
+    "build:clean": "tsc --build --clean && tsc --build",
+    "start:prod": "node build/index.js",
+    "start:dev": "ts-node __tests__/RestAPI.ts"
+  },
+  "dependencies": {
+    "@sphereon/ssi-express-support": "workspace:^",
+    "@sphereon/ssi-sdk.contact-manager": "workspace:^",
+    "@sphereon/ssi-sdk.data-store": "workspace:^",
+    "@sphereon/ssi-sdk-ext.key-manager": "0.14.0",
+    "@sphereon/ssi-sdk-ext.key-utils": "0.14.0",
+    "@sphereon/ssi-sdk.core": "workspace:^",
+    "@sphereon/ssi-sdk.kv-store-temp": "workspace:^",
+    "@sphereon/ssi-types": "workspace:^",
+    "@veramo/core": "4.2.0",
+    "body-parser": "^1.19.0",
+    "casbin": "^5.26.2",
```

Comment:
casbin and passport-http-bearer and morgan should move to dev deps

Reply:
moved